ops/services/app_service/main.tf

```diff
@@ -10,48 +10,87 @@ locals {
       "DOCKER_REGISTRY_SERVER_USERNAME" = data.terraform_remote_state.global.outputs.acr_simeplereport_name
       "WEBSITES_PORT"                   = "8080"
       "WEBSITE_DNS_SERVER"              = "168.63.129.16" # https://docs.microsoft.com/en-us/azure/app-service/web-sites-integrate-with-vnet#azure-dns-private-zones
-      "WEBSITE_VNET_ROUTE_ALL"          = "1"
   })
 }
 
-resource "azurerm_app_service_plan" "service_plan" {
+# migrate this to azurerm_service_plan.service_plan to enable migration of api service
+# resource "azurerm_app_service_plan" "service_plan" {
+#   name                = "${var.az_account}-appserviceplan-${var.env}"
+#   location            = var.resource_group_location
+#   resource_group_name = var.resource_group_name
+#   kind                = "linux"
+#   reserved            = true
+#   sku {
+#     tier = "PremiumV3"
+#     size = "P1v3"
+#   }
+# }
+
+resource "azurerm_service_plan" "service_plan" {
   name                = "${var.az_account}-appserviceplan-${var.env}"
   location            = var.resource_group_location
+  os_type             = "Linux"
   resource_group_name = var.resource_group_name
-
-  # Define Linux as Host OS
-  kind     = "Linux"
-  reserved = true
-
-  sku {
-    tier     = var.instance_tier
-    size     = var.instance_size
-    capacity = var.instance_count
-  }
+  sku_name            = "P2v3"
```

Comment:
what instances does this apply to? 
Most of our app services are on `P1v3`
except for
dev2 and stg are on `P2v3`
prod is on `P3v3`

Reply:
not sure why dev and dev2 is on P2v3 🤷

Reply:
Hm, I got it; a typo/oversight on my part; thanks for pointing this out.  I'll update this to be a variable per environment with a default of P1v3 but with prod passing in P3v3.

Reply:
This has been updated to a variable that has a default of P1V3, this is used for the API service